File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Java/src/test/java/gameCatan/factories/ComputerFactoryTest.java
package gameCatan.factories;

import gameCatan.models.Computer;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ComputerFactoryTest {

    @Test
    void create() {
        Computer computerTest = ComputerFactory.create();
        assertNotNull(computerTest);
    }
}

File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Kotlin/src/test/kotlin/gameCatan/factories/ComputerFactoryTest.kt
package gameCatan.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class ComputerFactoryTest {
    @Test
    fun create() {
        val computerTest = ComputerFactory.create()
        Assertions.assertNotNull(computerTest)
    }
}